src/components/academies/AcademyContentCard.jsx

```diff
@@ -123,18 +123,34 @@ const AcademyContentCard = ({
       return null;
     }
 
+    const getShowMoreBtnText = (contentTitle) => {
+      if (contentTitle === 'Self-paced courses') {
+        return `Show more ${title} (${contentLength}) >`;
+      }
+      return `Show more ${title} courses (${contentLength}) >`;
+    };
+
+    const getShowLessBtnText = (contentTitle) => {
+      if (contentTitle === 'Self-paced courses') {
+        return `< Show less ${title}`;
+      }
+      return `< Show less ${title} courses`;
+    };
+
     return (
       <div className={additionalClass}>
-        <div className="d-flex flex-row align-items-center justify-content-between">
-          <h3 data-testid={titleTestId}>{title}</h3>
+        <div className="d-flex flex-row align-items-center justify-content-between mt-5">
+          <h3 data-testid={titleTestId} className="font-weight-semi-bold">{title}</h3>
           {contentType !== LEARNING_TYPE_PATHWAY && contentLength > 4 && (
             <Button
               className=""
               variant="link"
               size="xl"
               onClick={toggleShowMore(contentType)}
             >
-              {showMoreButton(contentType) ? '< Show Less' : `Show More (${contentLength}) >`}
+              {showMoreButton(contentType)
```

Comment:
nit: I think it would be more readable to place this condition in a single line.

```
{ showMoreButton(contentType) ? getShowLessBtnText(title) : getShowMoreBtnText(title) }
```

Reply:
Also, can you consolidate these functions into a single function/hook? 

e.g.
```
const toggleButtonText = useMemo(() => {
  const enableShowMoreBtn = showMoreButton(contentType);
  if (enableShowMoreBtn) {
    return contentTitle === 'Self-paced courses'  ? `Show more ${title} (${contentLength}) >` : `Show more ${title} courses (${contentLength}) > `;
  } else {
    return contentTitle === 'Self-paced courses'  ? `< Show less ${title}` : `< Show less ${title} courses`;
}

}, [contentType , contentTitle, title, contentLength])
```